src/rpc/rpcquorums.cpp

```diff
@@ -649,42 +649,40 @@ static UniValue quorum_getdata(const JSONRPCRequest& request)
     });
 }
 
-static void quorum_getrotationinfo_help()
+static void quorum_rotationinfo_help()
 {
     throw std::runtime_error(
         RPCHelpMan{
             "quorum rotationinfo",
             "Get quorum rotation information\n",
-            {{"blockRequestHash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The blockHash of the request."},
-             {"baseBlockHashesNb", RPCArg::Type::NUM, RPCArg::Optional::NO,
-              "Number of baseBlockHashes"},
-             {"extraShare", RPCArg::Type::BOOL, RPCArg::Optional::NO, "Extra share"}},
+            {
+                {"blockRequestHash", RPCArg::Type::STR_HEX, RPCArg::Optional::NO, "The blockHash of the request"},
+                {"extraShare", RPCArg::Type::BOOL, /* default */ "false", "Extra share"},
+                {"baseBlockHash...", RPCArg::Type::STR_HEX, /* default*/ "", "baseBlockHashes"},
```

Comment:
Why is this argument name truncated?

Reply:
`...` just indicates that there are many of them. We could probably turn it into a json array instead if that would make it clearer.

Reply:
So it's just multiple strings separated by spaces? Is there a limit to how many can be put there?

Reply:
yes, it's multiple hashes. There is no limit atm (there was none in the code I refactored so I kept it). I'm not sure if we have to put one in place though.. Thoughts @ogabrielides @PastaPastaPasta ?